### PR TITLE
feat: tooltip summaries 返回原始数据

### PR DESCRIPTION
**Pull request by stone-lyl**
### 👀 PR includes

✨ Feature

- [X] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot
1. 添加原始数据的返回
2. 当 name 为空时，不显示总和。

| Before | After |
| ------ | ----- |
| ❌      | ✅     |
|  image | image |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/src/common/interface/tooltip.ts [hunk 1]
```diff
@@ -50,10 +50,13 @@ export interface TooltipOptions<Icon = Element | string, Text = string> {
...
 
+export type TooltipSummaryOptionsValue = number | string | undefined | null;
+
 export interface TooltipSummaryOptions {
-  name: string;
-  value: number | string;
+  name: string | null;
   selectedData: TooltipDataItem[];
+  value: TooltipSummaryOptionsValue;
+  originValue?: TooltipSummaryOptionsValue;
```
Comment (wjgogogo) [inline packages/s2-core/src/common/interface/tooltip.ts:59]: originValue是不是只可能是 number | undefined 哦

Comment (stone-lyl) [reply packages/s2-core/src/common/interface/tooltip.ts:59]: image
当为占位符时，也会返回 originValue ，所以也可能是其他类型

**Review by wjgogogo — APPROVED** [review]

**Pull request merged**